docs/intro/rucio.md

```diff
@@ -0,0 +1,7 @@
+# Scientific Data Management with Rucio
+
+[Rucio](https://rucio.cern.ch/) can be used to access experiments' data from SWAN. Rucio is a scientific data management system and comes with a JupyterLab extension that allows you to browse and download data from Rucio instances configured in the JupyterHub configuration.
+
+To access this extension, make sure you're using the lab instance configured with Rucio. You can check this by looking at the URL of the JupyterLab instance. It should contain the string `lab` instead of e.g. `tree`.
```

Comment:
Just silly comment, 
> You can check this by looking at the URL of the JupyterLab instance. It should contain the string `lab` instead of e.g. `tree`.

More than looking to the URL, the lab environment should be enable from the SWAN landing page.